module1/homework/removeVowels/vowel.cpp

```diff
@@ -0,0 +1,20 @@
+#include "vowel.hpp"
+#include <algorithm>
+
+
+void filterString(std::string &str, const std::string &filter) {
+
+    auto filterLetter = [&filter](const auto letter) {
+        return std::any_of(begin(filter), end(filter),
+                           [letter](auto vowel) { return vowel == std::tolower(letter); });
```

Comment:
Everything is nice but i have one thing - if you have this generic function that can use any filter - is there a reason to name the lambda parameter `vowel` :) ?

Reply:
right